get.sh

```diff
@@ -115,7 +115,13 @@ getBinaryOpenjdk()
 		fi
 	else 
 		download_url=$CUSTOMIZED_SDK_URL
-		wget -q --no-check-certificate --header 'Cookie: allow-download=1' ${download_url}
+		if [[ "$PLATFORM" == "s390x_zos" ]];
```

Comment:
We have added curl as a prerequisite for all platforms. So this condition could be removed and use curl for all cases.

Reply:
We should use a different PR to switch to using Curl in all cases.